app/test/globals.ts

```diff
@@ -5,6 +5,8 @@ const Dexie = require('dexie')
 Dexie.dependencies.indexedDB = require('fake-indexeddb')
 Dexie.dependencies.IDBKeyRange = require('fake-indexeddb/lib/FDBKeyRange')
 
+import 'airbnb-browser-shims/browser-only'
```

Comment:
:art: maybe a comment here to indicate what how this is useful, in case someone comes in later and wants to clean it up?

Reply:
a2a1be2